File: scala-gatling/src/gatling/scala/org/openapitools/client/model/RequestRoutes.scala

package org.openapitools.client.model


case class RequestRoutes (
    _locations: List[RequestLocation],
    _departureSearches: Option[List[RequestRoutesDepartureSearch]],
    _arrivalSearches: Option[List[RequestRoutesArrivalSearch]]
)
object RequestRoutes {
    def toStringBody(var_locations: Object, var_departureSearches: Object, var_arrivalSearches: Object) =
        s"""
        | {
        | "locations":$var_locations,"departureSearches":$var_departureSearches,"arrivalSearches":$var_arrivalSearches
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/RequestRoutes.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.RequestLocation
open OpenAPI.Model.RequestRoutesArrivalSearch
open OpenAPI.Model.RequestRoutesDepartureSearch

module RequestRoutes = 

  //#region RequestRoutes

  [<CLIMutable>]
  type RequestRoutes = {
    [<JsonProperty(PropertyName = "locations")>]
    Locations : RequestLocation[];
    [<JsonProperty(PropertyName = "departure_searches")>]
    DepartureSearches : RequestRoutesDepartureSearch[];
    [<JsonProperty(PropertyName = "arrival_searches")>]
    ArrivalSearches : RequestRoutesArrivalSearch[];
  }
  
  //#endregion
  